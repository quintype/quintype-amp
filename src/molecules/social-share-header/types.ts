export interface SocialShareHeaderProps {
  fbAppId?: string;
}
