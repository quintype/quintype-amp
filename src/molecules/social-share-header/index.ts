import { SocialShareHeader } from "./social-share-header";
export { SocialShareHeader };
