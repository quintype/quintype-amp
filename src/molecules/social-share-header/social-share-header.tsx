import React from "react";
import { SocialShareIcon, Spacer } from "../../atoms";
import styled from "styled-components";

const Wrapper = styled.div`
  display: flex;
`;

const SocialShareHeader = () => {
  const styles = { borderRadius: "50%", backgroundSize: "75%" };
  return (
    <Wrapper>
      <SocialShareIcon type="twitter" styles={styles} />
      <Spacer align="horizontal" token="m" />
      <SocialShareIcon type="facebook" styles={styles} />
      <Spacer align="horizontal" token="m" />
      <SocialShareIcon type="whatsapp" styles={styles} />
    </Wrapper>
  );
};

export { SocialShareHeader };
