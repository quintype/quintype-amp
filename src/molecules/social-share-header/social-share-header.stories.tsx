import * as React from "react";
import { storiesOf } from "@storybook/react";
import { SocialShareHeader } from "./social-share-header";
import { Layout } from "../../atoms";
import { textStory, config } from "../../__fixtures__";

storiesOf("SocialShareHeader", module)
  .addDecorator((story) => (
    <Layout story={textStory} config={config}>
      {story()}
    </Layout>
  ))
  .add("default", () => <SocialShareHeader />);
