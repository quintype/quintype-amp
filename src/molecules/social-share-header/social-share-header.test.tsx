import React from "react";
import { SocialShareHeader } from "./social-share-header";
import { shallow } from "enzyme";

describe("SocialShareHeader", () => {
  it("should render", () => {
    const wrapper = shallow(<SocialShareHeader />);
    expect(wrapper).toMatchSnapshot();
  });
});
