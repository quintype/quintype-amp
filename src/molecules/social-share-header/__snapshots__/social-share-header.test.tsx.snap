// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SocialShareHeader should render 1`] = `
<styled.div>
  <Component
    styles={
      Object {
        "backgroundSize": "75%",
        "borderRadius": "50%",
      }
    }
    type="twitter"
  />
  <styled.div />
  <Component
    styles={
      Object {
        "backgroundSize": "75%",
        "borderRadius": "50%",
      }
    }
    type="facebook"
  />
  <styled.div />
  <Component
    styles={
      Object {
        "backgroundSize": "75%",
        "borderRadius": "50%",
      }
    }
    type="whatsapp"
  />
</styled.div>
`;
